Specs/Scene/SunSpec.js

```diff
@@ -1,123 +1,90 @@
+    });
+
+    afterEach(function() {
+        scene.sun = undefined;
+    });
+
     function viewSun(camera, uniformState) {
         var sunPosition = uniformState.sunPositionWC;
-        camera.lookAt(sunPosition, Cartesian3.multiplyByScalar(Cartesian3.normalize(sunPosition, new Cartesian3()), CesiumMath.SOLAR_RADIUS + 100.0, new Cartesian3()));
+        var bounds = new BoundingSphere(sunPosition, CesiumMath.SOLAR_RADIUS);
+        camera.viewBoundingSphere(bounds);
     }
 
     it('draws in 3D', function() {
+        expect(scene.renderForSpecs()).toEqual(backgroundColor);
         scene.sun = new Sun();
-        scene.renderForSpecs();
+        scene.sun.glowFactor = 100;
+        scene.render();
 
         viewSun(scene.camera, scene.context.uniformState);
-        expect(scene.renderForSpecs()).not.toEqual([0, 0, 0, 0]);
+        expect(scene.renderForSpecs()).not.toEqual(backgroundColor);
     });
 
     it('draws in Columbus view', function() {
-        scene.sun = new Sun();
-
+        expect(scene.renderForSpecs()).toEqual(backgroundColor);
         scene.mode = SceneMode.COLUMBUS_VIEW;
-        scene.renderForSpecs();
+        scene.sun = new Sun();
+        scene.render();
 
         viewSun(scene.camera, scene.context.uniformState);
-        expect(scene.renderForSpecs()).not.toEqual([0, 0, 0, 0]);
+        expect(scene.renderForSpecs()).not.toEqual(backgroundColor);
     });
 
     it('does not render when show is false', function() {
-        var sun = new Sun();
-        sun.show = false;
-
-        var context = scene.context;
-        var frameState = createFrameState(context, createCamera({
-            near : 1.0,
-            far : 1.0e10
-        }));
-        var us = context.uniformState;
-        us.update(frameState);
-        viewSun(frameState.camera, us);
-        us.update(frameState);
-
-        scene._frameState = frameState;
-        var command = sun.update(scene);
-        expect(command).not.toBeDefined();
+        expect(scene.renderForSpecs()).toEqual(backgroundColor);
+        scene.sun = new Sun();
+        scene.render();
+        scene.sun.show = false;
 
-        sun.destroy();
+        viewSun(scene.camera, scene.context.uniformState);
+        expect(scene.renderForSpecs()).toEqual(backgroundColor);
     });
 
     it('does not render in 2D', function() {
-        var sun = new Sun();
-
-        var context = scene.context;
-        var frameState = createFrameState(context, createCamera({
-            near : 1.0,
-            far : 1.0e10
-        }));
-        frameState.mode = SceneMode.SCENE2D;
-        var us = context.uniformState;
-        us.update(frameState);
-        viewSun(frameState.camera, us);
-        us.update(frameState);
-        scene._frameState = frameState;
-        var command = sun.update(scene);
-        expect(command).not.toBeDefined();
-
-        sun.destroy();
-    });
-
-    it('does not render without a render pass', function() {
```

Comment:
I'm changing my mind here, but I think it's actually okay to include this test, but calling `sun.update(scene.frameState)` instead of `scene.renderForSpecs()`